src/object-cell.jsx

```diff
@@ -63,8 +63,18 @@ class ObjectCell extends React.Component {
   handleDoubleClick(event) {
     this.beginEdit();
   }
+  editable(objectId) {
```

Comment:
It doesn't matter much, coz I know it works, but will this also work?

```javascript
editable(objectId) {
  const { isReadOnly, editor } = this.props.column;
  const editorIsSet = !!editor; // bang bang! you're a boolean.
  const readOnly = typeof(isReadOnly) === 'function' ? isReadOnly(objectId) : isReadOnly;
  return editorIsSet && !readOnly;
}
```

Or, closer to your original code:

```javascript
editable(objectId) {
  const editorIsSet = this.props.column.editor !== false;
  let editable = editorIsSet;
  if (editorIsSet) {
    if (typeof(this.props.column.isReadOnly) === 'boolean') {
      editable = !this.props.column.isReadOnly;
    } else if (typeof(this.props.column.isReadOnly) === 'function') {
      editable = !this.props.column.isReadOnly(objectId);
    }
  }
  return editable;
}
```